lib/samlr/tools/response_builder.rb

```diff
@@ -95,9 +95,11 @@ def self.build(options = {})
 
         # The core response is ready, not on to signing
         response = builder.doc
+        assertion_options = options.merge(:skip_keyinfo => options[:skip_assertion_keyinfo])
+        response = sign(response, assertion_id, assertion_options) if sign_assertion
```

Comment:
Should we maybe make this assertion instead of response? just to keep it consistent?